pyiron_contrib/__init__.py

```diff
@@ -1,22 +1,25 @@
-from pyiron_contrib.image.job import ImageJob
-# from pyiron_contrib.protocol.generic import Protocol
-from pyiron_contrib.protocol.compound.minimize import ProtocolMinimize
-from pyiron_contrib.protocol.compound.molecular_dynamics import ProtocolMD
-from pyiron_contrib.protocol.compound.nudged_elastic_band import ProtocolNEB
-# from pyiron_contrib.protocol.compound.tild import HarmonicTILD, VacancyTILD
-# from pyiron_contrib.protocol.compound.finite_temperature_string import StringRelaxation, VirtualWork, \
-#     Milestoning, VirtualWorkParallel, VirtualWorkSerial, VirtualWorkFullStep
-from pyiron_contrib.protocol.compound.qmmm import ProtocolQMMM
-from pyiron_contrib.protocol.compound.tild import ProtocolHarmonicTILD, ProtocolHarmonicTILDParallel
+from pyiron import Project
+from pyiron.base.job.jobtype import JOB_CLASS_DICT
 
-# protocol is a magic class after this one we have to Register
-# from pyiron_contrib.protocol.utils.types import PyironJobTypeRegistryMetaType
-# PyironJobTypeRegistryMetaType.inject_dynamic_types()
-__all__ = [
-    'ImageJob',
-    'ProtocolMinimize',
-    'ProtocolMD',
-    'ProtocolNEB',
-    'ProtocolQMMM',
-    'ProtocolHarmonicTILD', 'ProtocolHarmonicTILDParallel',
-]
+# Make classes available for new pyiron version
+JOB_CLASS_DICT['ProtocolMinimize'] = 'pyiron_contrib.protocol.compound.minimize'
+JOB_CLASS_DICT['ProtocolMD'] = 'pyiron_contrib.protocol.compound.molecular_dynamics'
+JOB_CLASS_DICT['ProtocolNEB'] = 'pyiron_contrib.protocol.compound.nudged_elastic_band'
+JOB_CLASS_DICT['ProtocolQMMM'] = 'pyiron_contrib.protocol.compound.qmmm'
+JOB_CLASS_DICT['ProtocolHarmonicTILD'] = 'pyiron_contrib.protocol.compound.tild'
+JOB_CLASS_DICT['ProtocolHarmonicTILDParallel'] = 'pyiron_contrib.protocol.compound.tild'
+
+# Backwards compatibility
```

Comment:
Should not this part be in `pyiron_mpie` ?

Reply:
I agree. At times I only import `pyiron_contrib` and not `pyiron_mpie`. This kind-of helps in those circumstances. However, I can remove it. Shall I?

Reply:
I would prefer that, because people who do not have access to `pyiron_mpie` might be confused.

Reply:
Sure. I shall remove it.